_drafts/zio-kafka.md

```diff
@@ -0,0 +1,649 @@
+---
+title: "Zio Kafka: A Practical Example of Zio Streams"
+date: 2021-07-24 
+header:
+  image: "/images/blog cover.jpg"
+tags: []
+excerpt: "Modern distributed applications need a communication system between their components that must be reliable, scalable, and efficient. Apache Kafka is a message broker that in the last years proved to have these features. What's the best way to interact with such a message broker, if not with the ZIO ecosystem?"
+---
+
+Modern distributed applications need a communication system between their components that must be reliable, scalable, and efficient. Synchronous communication based on HTTP is not a choice in such applications due to latency problems, insufficient resources' management, etc... Hence, we need an asynchronous messaging system capable of quickly scaling, robust to errors, and low latency.
+
+Apache Kafka is a message broker that in the last years proved to have the above features. What's the best way to interact with such a message broker, if not with the ZIO ecosystem? Hence, **ZIO provides an asynchronous, reliable, and scalable programming model, which perfectly fits the feature of Kafka**. So, let's proceed without further ado.
+
```

Comment:
Add a small sentence that describes what we're going to cover in the article - which features of Kafka + ZIO, and what the use-case is (e.g. a real-time notificator for a football tournament).